src/bls/bls_aggregator.cpp

```diff
@@ -32,8 +33,31 @@
 
 namespace eth {
 
+// When a service node receives a request to sign an exit request for a BLS node, this values
+// determines how old that request is allowed to be from the time it was initiated to us receiving
+// it, that we will consider signing it.
+inline constexpr auto BLS_MAX_TIME_ALLOWED_FOR_EXIT_REQUEST = 3min;
```

Comment:
Perhaps also rename to:
```suggestion
constexpr auto BLS_EXIT_REQUEST_MAX_AGE = 3min;
```

Reply:
Done, that's a better name.